appstream.patch

```diff
@@ -0,0 +1,77 @@
+From 9f718c7b688bb5d7de4f4934b6d0b8cdb37c5823 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?= <matthias@mailaender.name>
+Date: Mon, 12 Dec 2022 14:36:33 +0000
+Subject: [PATCH 1/3] Add Open Age Rating
+
+---
+ org.kde.SymbolEditor.appdata.xml | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/org.kde.SymbolEditor.appdata.xml b/org.kde.SymbolEditor.appdata.xml
+index b2fd9bb..bffeba6 100644
+--- a/org.kde.SymbolEditor.appdata.xml
++++ b/org.kde.SymbolEditor.appdata.xml
+@@ -259,6 +259,7 @@
+       <li xml:lang="x-test">xxSymbol creation from charactersxx</li>
+     </ul>
+   </description>
++  <content_rating type="oars-1.1"/>
+   <url type="homepage">https://userbase.kde.org/Special:myLanguage/SymbolEditor</url>
+   <url type="bugtracker">https://bugs.kde.org/enter_bug.cgi?format=guided&amp;product=SymbolEditor</url>
+   <url type="help">http://docs.kde.org/?application=SymbolEditor</url>
+-- 
+GitLab
+
+
+From a10ebb826653aaf3222b5dec43aea1a5a79936cb Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?= <matthias@mailaender.name>
+Date: Mon, 12 Dec 2022 14:38:14 +0000
+Subject: [PATCH 2/3] Add releases
+
+---
+ org.kde.SymbolEditor.appdata.xml | 5 +++++
+ 1 file changed, 5 insertions(+)
+
+diff --git a/org.kde.SymbolEditor.appdata.xml b/org.kde.SymbolEditor.appdata.xml
+index bffeba6..bfcc923 100644
+--- a/org.kde.SymbolEditor.appdata.xml
++++ b/org.kde.SymbolEditor.appdata.xml
+@@ -272,4 +272,9 @@
+   <provides>
+     <binary>SymbolEditor</binary>
+   </provides>
++  <releases>
++    <release version="2.1.0" date="2019-06-30"/>
++    <release version="2.0.0" date="2015-11-16"/>
++    <release version="1.5.0" date="2014-08-17"/>
++  </releases>
+ </component>
+-- 
+GitLab
+
+
+From 76fef92b68cdf647b4dcdf400ddfb6141d007571 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?= <matthias@mailaender.name>
+Date: Mon, 12 Dec 2022 15:31:07 +0000
+Subject: [PATCH 3/3] Use a more permissive metadata license.
+
+---
+ org.kde.SymbolEditor.appdata.xml | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/org.kde.SymbolEditor.appdata.xml b/org.kde.SymbolEditor.appdata.xml
+index bfcc923..3370c3b 100644
+--- a/org.kde.SymbolEditor.appdata.xml
++++ b/org.kde.SymbolEditor.appdata.xml
+@@ -1,7 +1,7 @@
+ <?xml version="1.0" encoding="utf-8"?>
+ <component type="desktop">
+   <id>org.kde.SymbolEditor.desktop</id>
+-  <metadata_license>GFDL-1.2-without-invariant</metadata_license>
++  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
you can't do that. You have no rights to change somebody else license. You should contact upstream and ask them to relicense the file under an acceptable license.

or write a new one from scratch.

Reply:
I submitted it as a patch. https://invent.kde.org/graphics/symboleditor/-/merge_requests/4

Reply:
that doesn't make it valid. It's a **license change** and only the rights-holder can agree to it, at least in that direction (CC0 is more permissive)

Reply:
Yes, I don't understand why you blacklist the valid GFDL one here. We now have to wait for all authors to agree and the pull request to be merged.

Reply:
Out of curiosity, have you checked if libappstream tooling would complain about it?

Reply:
Upstream rejected the license change.

Reply:
We did not "reject the license change" but we can not just change a license discretely in a PR with other changes. There is a procedure to follow: It needs to be an independent PR with just that change and that asks all authors that contributed to the file if they are OK with the change.

Reply:
I suggest a less invasive change now: https://invent.kde.org/graphics/symboleditor/-/merge_requests/5